ARCHITECTURE.md

```diff
@@ -51,34 +51,44 @@ https://github.com/replicatedhq/kurl-api
 
 The browser UI for creating and viewing kURL installers.
 
-#### Go API Proxy
+#### Typescript API
 
-This API proxies most requests directly to the Typescript API.
-Additionally, this API interacts with the object store and is responsible for assembling and streaming air-gap bundles back to the end-user.
+https://github.com/replicatedhq/kURL-api/tree/main/src
 
-#### Typescript API
+The API for creating kURL URLs and rendering installers.
+The API accepts a kURL Installer spec and returns a URL including a deterministic hash of the spec for installing a cluster based on that spec, for example https://k8s.kurl.sh/5e61e80.
+The API interacts stores these hashes in its relational database for retrieval.
```

Comment:
That still not clear for me. We have all possible combinations for each release stored. Am I right?
Then, how/where the SHA is generated? is possible to link the code / place?
Also, what is the schema of this table? 
Is this scripts also stored in the same database that we have the replicated users, portal data or have we an database only for kURL? I  understand that have all together. 

But I think we need to have a place that describe how it is working as pointing out. If we be unable to put here then, wdyt about we add in the wiki? Then we link the wiki page here ( valid only for replicated team members something like that)

Reply:
Understand what we store, when and how as where it is used help us to understand how the project is working, the requirements, how the projects are connected and etc. I still a little strangle to get this part.

Reply:
I've tried to address all these questions in the commit https://github.com/replicatedhq/kURL/pull/3749/commits/3548f35b19cf194a5415a232e21b5b26b8a9c5f7